src/assets/docker-entrypoint.sh

```diff
@@ -132,6 +134,25 @@ if (echo "${ACTIVEMQ_ARTEMIS_VERSION}" | grep -Eq  "(1.5\\.[3-5]|[^1]\\.[0-9]\\.
   performanceJournal
 fi
 
+###
+# Update dynamic env vars
```

Comment:
This coment style is different to the comment style of the rest of the file.

Reply:
updated